### PR TITLE
Klarocca/side menu

### PR DESCRIPTION
**Pull request by kimlarocca**
Added an option to not have the side menu in the header like we discussed

### CODE REVIEW

File: addon/templates/components/nypr-o-header.hbs [hunk 1]
```diff
@@ -2,14 +2,25 @@
...
 
-  {{yield (hash
-    menu=(component 'nypr-o-header/menu' donateURL=@donateURL close=(action toggleMenu false) aria-hidden=(if this.isOpen 'false' 'true') isOpen=this.isOpen)
-    left=(component 'nypr-o-header/left' openMenu=(action toggleMenu) ariaExpanded=(if this.isOpen 'true' 'false') ariaLabel=(if this.isOpen 'Close navigation menu' 'Open navigation menu'))
-    nav=(component 'nypr-o-header/nav' class="c-primary-nav--top" linkDataLabel='Top Nav')
-    right=(component 'nypr-o-header/right')
-    outOfViewport=this.outOfViewport
-    rules=this.rules
-  )}}
+  {{#if @showMenuButton}}
```
Comment (walsh9) [inline addon/templates/components/nypr-o-header.hbs:5]: If you put this conditional into the `nypr-o-header/left.hbs` template instead, you won't need to use two separate components or duplicate this block

File: stories/90-Components/Organisms/Site Header.stories.mdx [hunk 2]
```diff
@@ -393,3 +393,101 @@ On mobile, the majority of components in the Site Header will be hidden as to fi
...
+<Preview withToolbar={true}>  
+  <Story name="Header Config 4 (No Menu)" height="150px">{{
+    template: hbs`
+          <NyprOHeader
+                  @showMenuButton=false
+                  @donateURL={{DONATE_URL}}
+                  data-test-main-header
+                  as |header|
+          >
+            <header.left data-test-header-left as |left|>
```
Comment (walsh9) [inline stories/90-Components/Organisms/Site Header.stories.mdx:406]: With the change suggested above this part of the example would become
`<header.left @showMenuButton=false as |left|>`

**Review by walsh9 — CHANGES_REQUESTED** [review]

File: stories/90-Components/Organisms/Site Header.stories.mdx (hunk 2 above)
Comment (kimlarocca) [reply stories/90-Components/Organisms/Site Header.stories.mdx:406]: If we make this change, we'd have to set the showMenuButton=true or =false for all the headers

Comment (walsh9) [reply stories/90-Components/Organisms/Site Header.stories.mdx:406]: We can have a default, the same as if we set it in the top level.

**Review by walsh9 — APPROVED** [review]

**Pull request merged**